From b87c584e1e62d0e5bcbaa8895f73cee85fc8cb72 Mon Sep 17 00:00:00 2001
From: Ranjani Vaidyanathan-RA5478 <Ranjani.Vaidyanathan@freescale.com>
Date: Tue, 21 Oct 2008 10:09:14 -0500
Subject: [PATCH] ENGR00095623-2 regulator: Change SW1 regulator constraints

Fix the min and max voltage values for SW1 regulator.

Signed-off-by: Ranjani Vaidyanathan-RA5478 <Ranjani.Vaidyanathan@freescale.com>
---
 drivers/regulator/mc13892/reg-mc13892.c |    8 +++++++-
 1 files changed, 7 insertions(+), 1 deletions(-)

diff --git a/drivers/regulator/mc13892/reg-mc13892.c b/drivers/regulator/mc13892/reg-mc13892.c
index 951276e..96f9c0b 100644
--- a/drivers/regulator/mc13892/reg-mc13892.c
+++ b/drivers/regulator/mc13892/reg-mc13892.c
@@ -1426,6 +1426,12 @@ static struct regulator_ops mc13892_sw_ops = {
 	.get_voltage = mc13892_sw_get_voltage,
 };
 
+struct regulation_constraints sw1_constraints = {
+	.min_uV = mV_to_uV(600),
+	.max_uV = mV_to_uV(1375),
+	.valid_ops_mask = REGULATOR_CHANGE_VOLTAGE,
+};
+
 struct regulation_constraints sw_constraints = {
 	.min_uV = mV_to_uV(1100),
 	.max_uV = mV_to_uV(1850),
@@ -1608,7 +1614,7 @@ static struct mc13892_regulator reg_mc13892[] = {
 		       .name = "SW1",
 		       .id = MC13892_SW1,
 		       .ops = &mc13892_sw_ops,
-		       .constraints = &sw_constraints,
+		       .constraints = &sw1_constraints,
 		       },
 	 },
 	{
-- 
1.5.4.4

